browse/routes/ui.py

```diff
@@ -187,6 +187,47 @@ def list_articles(context: str, subcontext: str) -> Response:
         return response, code, headers  # type: ignore
 
 
+@blueprint.route('stats/<string:command>',
+                 methods=['GET'])
+def stats(command: str) -> Response:
+    """Display various statistics about the service."""
+    params: Dict = {}
+    if request.args and 'date' in request.args:
+        params['requested_date_str'] = str(request.args['date'])
+
+    getters: Mapping[str, Mapping[str, Union[Callable, Union[Dict, Mapping]]]] = {
```

Comment:
By doing the path to function mapping inside of the stats() function the blueprint/flask decorator route mapping capability is being redundantly reproduced. This will affect the regularity and predictability of our use of Flask/blueprints. Ex. url_for() will not be as useful.

You may want to split these out to something like:

@blueprint.route('stats/monthly_downloads')
def...
@blueprint.route('stats/today')
def...
@blueprint.route('stats/monthly_submissions')
def...
@blueprint.route('stats/get_monthly_downloads')
def...
@blueprint.route('stats/get_hourly')
def...
@blueprint.route('stats/get_monthly_submissions')
def...

Reply:
I was torn on this, but had a hard time deciding which option is less ugly. This is also an option:
```
@blueprint.route('stats/monthly_downloads')
@blueprint.route('stats/today')
@blueprint.route('stats/monthly_submissions')
@blueprint.route('stats/get_monthly_downloads')
@blueprint.route('stats/get_hourly')
@blueprint.route('stats/get_monthly_submissions')
def stats(...)
```
I'll look at it again.

Reply:
Going to leave this now. May resurface for release/team PR.